README.md

```diff
@@ -7,13 +7,42 @@ Boilerplate to create an embedded Shopify app made with Node, [Polaris](https://
 
 ## Installation
 
-Using the [Shopify-App-CLI](https://github.com/Shopify/shopify-app-cli) run:
+Using the [Shopify CLI](https://github.com/Shopify/shopify-cli) run:
 
 ```sh
-~/ $ shopify node create -n APP_NAME
+shopify app create node -n APP_NAME
 ```
 
-Or, fork and clone repo
+Or, you can clone this repo and create a `.env` file containing the following values:
```

Comment:
```suggestion
Or, you can run `npx degit shopify/shopify-app-node` and create a `.env` file containing the following values:
```

This will give a git history free version of the repo :)

Reply:
Neat, TIL. Although it's a little surprising that it creates at at your current path and not in a new `shopify-app-node` directory that `git clone` would.